spec/lib/importers/rss_feed_data_spec.rb

```diff
@@ -402,7 +415,7 @@
         stub_request(:get, rss_feed_url.url).to_return({ status: 200, body: rss_feed_content })
       end
 
-      it 'should not change the number of news items, and update the crawl status' do
+      it 'does not change the number of news items, and update the crawl status' do
```

Comment:
Should be "does not change the number of news items, and updates the crawl status"